tests/recipes/test_eleuther_eval.py

```diff
@@ -231,7 +231,7 @@ def test_meta_eval_vision(self, caplog, monkeypatch, tmpdir, expected_vision_acc
             checkpointer.model_type=LLAMA3_VISION \
             tokenizer.path=/tmp/test-artifacts/tokenizer_llama3.model \
             tokenizer.prompt_template=null \
-            limit=4 \
+            limit=1 \
```

Comment:
At a glance, I'm worried this isn't a good enough signal for changes in the recipe due to the fact that the majority of MMMU is multiple choice. 

I'd postulate (ha big word) that a minimum of 3 is good, but I'd like to understand the tradeoff better. What are the run times for 1 vs. 2 vs. 3 vs. 4? Also what are the questions for each subject that come up? Are the open or mc?

Reply:
Okay it's not coming up anymore so I'm going slightly insane. 

> Also what are the questions for each subject that come up? Are the open or mc?

They're all MC questions I think.

> What are the run times for 1 vs. 2 vs. 3 vs. 4? 

The runtime for `limit=1` was around 80s, so I'd _postulate_ that 2 is 160s, 3 is 240s, etc.